src/ontology/cl-edit.owl

```diff
@@ -27689,6 +27690,19 @@ SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002162 obo:NCBITaxon_1009
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534))
 SubClassOf(obo:CL_4023036 ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_P32848))
 
+# Class: obo:CL_4023041 (L5 extratelencephalic projecting glutamatergic cortical neuron)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2020.10.19.343129") obo:IAO_0000115 obo:CL_4023041 "A glutamatergic neuron, found in the deeper portion of L5, that have long-range axonal projections that include deep subcortical targets outside of the telencephalon that can include the spinal cord. While the L5 ET neuron projections are not limited to ET targets, they are clearly differentiated from the neuron subclasses whose projections are constrained to intratelencephalic (IT) targets.  L5 ET neurons are generally the largest excitatory cortical neurons, typically having a thick apical dendrite with a prominent dendritic tuft in layer 1 and displaying burst-firing physiological characteristics.")
```

Comment:
Some minor edits to def text:

"A glutamatergic neuron, found in the deeper portion of L5, that has~~ve~~ long-range axonal projections ~~that~~ including~e~ deep subcortical targets outside of the telencephalon ~~that~~ and, in some cases, the spinal cord. While the L5 ET neuron projections are not limited to ET targets, they are clearly differentiated from the neuron subclasses whose projections are constrained to intratelencephalic (IT) targets.  L5 ET neurons are generally the largest excitatory cortical neurons, typically having a thick apical dendrite with a prominent dendritic tuft in layer 1 and displaying burst-firing physiological characteristics."